pdal/util/FileUtils.hpp

```diff
@@ -85,6 +85,14 @@ namespace FileUtils
     */
     PDAL_DLL bool createDirectory(const std::string& dirname);
 
+    /**
+      Create a set of directories.
+
+      \param dirname  Directory name.
+      \return  Whether the directory was created.
+    */
+    PDAL_DLL bool createDirectories(const std::string& dirname);
```

Comment:
Document this better.  The arg is a string, but this represents multiple directories - how is this string formatted?  To me a `std::vector<std::string>` or a `set` would be a better input.

Reply:
I'm not sure that I changed this...

Reply:
This is something I did for SlpkExtractor().  It just makes all the directories necessary to complete a path.  I'll make the description better.